QUARKUS-5614

```diff
@@ -0,0 +1,66 @@
+# QUARKUS-5614 Support @PermissionsAllowed defined on meta-annotation
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5614
+
+Support @PermissionsAllowed defined on meta-annotation
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/43241
+
+Upsteram issue:
+- https://github.com/quarkusio/quarkus/issues/39663
+
+Documentation:
+- https://quarkus.io/guides/security-authorize-web-endpoints-reference
+
+## Scope of testing
+Verify that `@PermissionsAllowed` works correctly when applied on a meta-annotation.
+These tests will cover various scenarios of meta-annotations defined on both methods and classes.
+Furthermore, the tests will validate the combined usage of directly applied `@PermissionsAllowed` annotations and those defined through meta-annotations on a single method, ensuring they operate together.
+Additionally, the direct application of `@PermissionsAllowed` on methods and classes will be tested, as this is currently not covered by QE.
+
+
+## Automated test development
+
+### Definition of roles and permissions
+
+The tests will use 4 roles (for 4 users) and 4 defined permission.
+Each role will be assigned a unique set of permissions.
+These roles will be defined in property file using Elytron security.
+The need for 4 roles and 4 permission is to validate multiple combination and definition.
+
+### Test cases
+- Both definition on meta-annotation and direct use of `@PermissionsAllowed` will have same cases, which should have same results.
+- These cases are:
+  - Two annotation on one method where each have one permission
+  - Two annotation on one method where one have one permission and second have multiple permission
+  - Two annotation on one method where one have one permission and second have multiple permission and defined inclusive (user role must have all permission)
+  - One annotation with multiple permission
+  - One annotation with multiple permission and defined inclusive
+  - Permission inherted from definition in interface
```

Comment:
if you read https://quarkus.io/guides/security-authorize-web-endpoints-reference#endpoint-security-annotations-and-jakarta-rest-inheritance than you know that while historically some scenarios work (maybe we could say even many), inheritance from interface is not supported.

Reply:
yeah and there is difference between annotations on endpoints and annotations on CDI beans that are not endpoints, again, I am fine if you are going to only test endpoints

Reply:
Thanks I missed this one part of docs. I removed these usecases